### DISCUSSION TITLE
Reservations are showing as available IPs

### DISCUSSION THREAD

**Discussion by BartBaljon** [netbox-community/netbox#17996] [General]
Hello Netbox group,

I have been working towards documenting our organization into Netbox and move away from spreadsheets. Thank you for this awesome tool, it's been amazing.

I have come across something today that I am unsure if this is by design or an issue within Netbox.

To start off, I've created a prefix (10.110.150.0/26). This prefix has a parent prefix at 10.110.0.0/16.

10.110.150.1 will be my gateway.
10.110.150.2-.9 will be reserved.
My next available IP should be 10.110.150.10.

![Netbox1](https://github.com/user-attachments/assets/eadea1a6-e150-4e85-8c3d-376f16c8bba0)

I have created an IP Range under this prefix, which is seen below:
![Netbox2](https://github.com/user-attachments/assets/b797f1d8-4e7a-4baa-a878-d98f9de4690e)

Now, when I go into the IP addressing of this prefix, if I click on the **Add IP Address** button, Netbox correctly prompts me that the next IP to use is: **10.110.150.10/26**

But if I click on the **61 IPs available** button, Netbox will offer me the following IP address to use: **10.110.150.2/26**
![Netbox3](https://github.com/user-attachments/assets/709abbc1-f794-48d3-8f7a-23e1b1e761eb)

I am curious if this is by design? I was hoping that once I set the IPs in the IP range as reserved, that Netbox would be able to adjust accordingly and not show "61 IPs available". I understand that they may be available but I thought that having them reserved would remove them from being seen as available.

Thank you,

**Comment by candlerb** [comment]
IP Ranges and IP Addresses in Netbox are independent objects. There's no unified view which shows both, which would be great, but it's been a limitation since IP Ranges were added to Netbox.

See for example #6934, #9763
